src/app/qgisapp.cpp

```diff
@@ -3544,6 +3545,19 @@ void QgisApp::refreshProfileMenu()
   QAction *newProfileAction = mConfigMenu->addAction( tr( "New Profile…" ) );
   newProfileAction->setObjectName( "mActionNewProfile" );
   connect( newProfileAction, &QAction::triggered, this, &QgisApp::newProfile );
+
+  mConfigMenu->addSeparator( );
+
+  QAction *openProfileSelectionDialog = mConfigMenu->addAction( tr( "Open Profile Selector" ) );
```

Comment:
```suggestion
  QAction *openProfileSelectionDialog = mConfigMenu->addAction( tr( "Open Profile Selector…" ) );
```

As a dialog will be opened after the click and the user will need to interact with it.

Reply:
Good point! Added in 5eb0a50442444acb8cfb542bb97a904badcfeabc